test/e2e/apimachinery/webhook.go

```diff
@@ -1659,17 +1629,6 @@ func namedNonCompliantConfigMap(name string, f *framework.Framework) *v1.ConfigM
 	}
 }
 
-func nonDeletableConfigmap(f *framework.Framework) *v1.ConfigMap {
-	return &v1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: nonDeletableConfigmapName,
```

Comment:
nonDeletableConfigmapName is unused now, right?

Reply:
Yes, you are right.
Thanks for your suggestion.
I update this PR.